engine/src/main/java/org/destinationsol/ModuleManager.java

```diff
@@ -94,6 +94,7 @@ public ModuleManager() {
         requiredModules.add(registry.getLatestModuleVersion(new Name("core")));
         requiredModules.add(registry.getLatestModuleVersion(new Name("federal")));
         requiredModules.add(registry.getLatestModuleVersion(new Name("organic")));
+	requiredModules.add(registry.getLatestModuleVersion(new Name("formic")));
```

Comment:
Whitespace is off here. Make sure you're using IntelliJ/Atom with comparable settings (some sort of smart adaption where it'll match existing whitespace or hard set to 4-space indentation)